package.json

```diff
@@ -99,7 +99,7 @@
     "@hot-loader/react-dom": "^16.13",
     "@reach/auto-id": "^0.13.0",
```

Comment:
"Upload thumbnail" modal is transparent -- hard to read.
<img src="https://user-images.githubusercontent.com/64950861/149901717-46d94fc5-dc0b-4763-aac8-c6fa6924d991.png" width="300">

Sometimes, it's not transparent -- inconsistent.  Not sure how to reproduce, though.  Seems random.
<img src="https://user-images.githubusercontent.com/64950861/149902105-5b515704-98e4-4a87-afbe-111839162271.png" width="300">

Reply:
Fixed, but not yet tested in every form. The background setting was one level deeper than the modal (.card) and some of the modals don't use the card class. That's why this one had no background in the first screenshot. The reason why it seemed inconsistant was the focus selector. The purple background & border got applied when the element was focused. Not random, but irritating and wrong. Thanks :D